### ISSUE TITLE
Multiple device unread count mismatch

### ISSUE THREAD

**Issue by Xefir** [CarlosEsco/Neko#1290]
### Steps to reproduce

On device one : 
- Log in to MangaDex and enable synchronisation
- Read any manga but not until the end

One device two :
- Log in to MangaDex enable synchronisation
- Sync MangaDex Follows
- Read the same manga a little more, but not until the end

One device one : 
- Refresh the library and wait until the end
- Unread count isn't refresh on this manga and the old unread count is synced automatically on MangaDex

### Expected behavior

The unread count of device two (and so the unread count stored on MangaDex) should be pulled on refresh

### Actual behavior

The old unread count of device one is uploaded on MangaDex, erasing the good unread count set up by the device two

### Neko version

2.12.3

### Android version

Android 11

### Device

Nexus 7 and OnePlus 9

### Other details

Maybe I doing wrong and I miss something but I just would like to be able to resume my reads on any device with Neko and MangaDex logged in with the same MandaDex account.

Thank you :)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by Xefir** [comment]
Hum, after doing it again, it seems the problem is elsewhere.
I just notice that I was kicked out from MangaDex on device one when I log in on device two and vice-versa.
So when I refresh, It does not have the MangaDex account linked to do the right thing.

Sorry for the issue, after login back it works (except I have to log in again on each device each time, but now that I know that, I can sync my reads between my device :) )

**Comment by CarlosEsco** [comment]
Yeah. Until the mangadex refresh token issue is resolved on their end, you're going to need a daily login if not more often since it's a 7 hour expiration
